pkg/ingester/ingester_ring.go

```diff
@@ -13,12 +13,16 @@ import (
 	"github.com/grafana/dskit/kv"
 	"github.com/grafana/dskit/netutil"
 	"github.com/grafana/dskit/ring"
+
+	"github.com/grafana/mimir/pkg/util"
 )
 
 const (
 	// sharedOptionWithRingClient is a message appended to all config options that should be also
 	// set on the components running the ingester ring client.
-	sharedOptionWithRingClient = " This option needs be set on ingesters, distributors, queriers and rulers when running in microservices mode."
+	sharedOptionWithRingClient   = " This option needs be set on ingesters, distributors, queriers and rulers when running in microservices mode."
+	prefix                       = "ingester.ring."
```

Comment:
[nit] `prefix` is a very generic name for being a global variable. I would clarify it:

```suggestion
	cliFlagPefix                       = "ingester.ring."
```